robocop/run.py

```diff
@@ -130,16 +132,20 @@ def any_rule_enabled(self, checker):
 
     def configure_checkers(self):
         for config in self.config.configure:
-            # TODO: handle wrong format, not existing checker
+            if config.count(':') != 2:
+                print(f'Provided invalid config: \'{config}\' (pattern: <rule>:<param>:<value>)')
+                continue
             rule, param, value = config.split(':')
             if rule not in self.messages:
+                print(f'Provided rule \'{rule}\' does not exists')
```

Comment:
Or is disabled? Im not sure if disabled rules are still kept in messages or not

Reply:
`self.messages` contains all messages. You need to call `self.config.is_rule_enabled(msg)` to see which is enabled. I can extend the functionality or add new cli option to list all enabled/disabled messages.

Reply:
It would be handy for verbose output (printing rules with disabled/enabled flag before scan).